index.html

```diff
@@ -895,13 +895,13 @@ <h3 id="event-touchcancel">The <dfn class="event"><code>touchcancel</code></dfn>
       </p>
 
       <p class="note">
-        User agents will typically dispatch mouse and click events when there is only a single
-        <a>active touch point</a>. Multi-touch interactions – involving two or more
+        User agents will typically dispatch mouse and click events only for single finger activation gestures (like tap and long press), and only when there is a single
+        <a>active touch point</a>. Gestures involving movement of the touch point or multi-touch interactions – with two or more
         <a href="#dfn-active-touch-point">active touch points</a> – will usually only generate touch events.
       </p>
 
       <p id="click-events">
-        If the user agent interprets a sequence of touch events as a click,
+        If the user agent interprets a sequence of touch events as a single finger activation gestures,
```

Comment:
"tap gesture" is probably what you want here, right?  Long press is similar but generated "contextmenu" instead of "click" (and is probably not worth spelling out explicitly).